lib/components/STable.vue

```diff
@@ -20,6 +21,11 @@ import STableItem from './STableItem.vue'
 
 const props = defineProps<{
   options: Table
+  selected?: number[]
```

Comment:
Can we make this support `string[]` too? The unique identifier could be `string` in some cases 👀